code42_connector.py

```diff
@@ -64,8 +61,11 @@ def _handle_test_connectivity(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
         self.save_progress("Connecting to endpoint")
 
-        response = self._client.users.get_current()
-        util.print_response(response)
+        try:
+            self._client.users.get_current()
+        except Exception as e:
```

Comment:
Not sure what style we are supposed to follow, but PEP8 recommends avoiding single character variable names.

Reply:
they have some sort of linter built into their compile utility. I'm going to try to see if we can actually get it to run as part of CI.